cetmix_tower_aws/readme/DESCRIPTION.md

```
@@ -0,0 +1,2 @@
+This module integrates Boto3, the Amazon Web Services (AWS) Software Development Kit (SDK) for Python, into the Cetmix Tower.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)